code/obj/item/tool/omnitool.dm

```diff
@@ -1,3 +1,9 @@
+TYPEINFO(/obj/item/tool/omnitool)
+	mats = list("dense_super" = 20,
+				"conductive_high" = 20,
+				"insulated_property" = 15,
+				"miracle" = 10,)
```

Comment:
Miracle is usually for things that produce matter out of nothing (chem dispensers etc.), seems excessive here.